x/payment/types/params.go

```diff
@@ -0,0 +1,99 @@
+package types
+
+import (
+	"fmt"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v2"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+var (
+	KeyMinSquareSize = []byte("MinSquareSize")
+	// TODO: Determine the default value
+	DefaultMinSquareSize int32 = 0
+)
+
+var (
+	KeyMaxSquareSize = []byte("MaxSquareSize")
+	// TODO: Determine the default value
+	DefaultMaxSquareSize int32 = 0
+)
+
+// ParamKeyTable the param key table for launch module
```

Comment:
[question] what is launch in this context? Perhaps:

```suggestion
// ParamKeyTable the param key table for the payment module
```